suite-native/module-home/src/components/DashboardHeaderDeviceChip.tsx

```diff
@@ -27,7 +27,7 @@ export const DashboardHeaderDeviceChip = () => {
                 onPress={() => setIsDeviceModalVisible(true)}
             >
                 <Icon name="trezorT" />
-                <Text>Trezor T</Text>
+                <Text>Trezor Model T</Text>
                 <Box>
                     <Icon name="chevronUp" size="small" />
                     <Icon name="chevronDown" size="small" />
```

Comment:
This file is unused. Fixed here https://github.com/trezor/trezor-suite/pull/8911

Reply:
commit skipped